### ISSUE TITLE
figure out how we can implement gradient normalization for loss functions

### ISSUE THREAD

**Issue by povidllo** [povidllo/Calculation-module-based-on-PINN-neural-network-architecture#25]
https://arxiv.org/pdf/1711.02257
article about it

https://github.com/LucasBoTang/GradNorm

https://github.com/lucidrains/gradnorm-pytorch
